test/integration/dualstack/dualstack_test.go

```diff
@@ -660,6 +715,14 @@ func TestCreateServiceDualStackIPv4IPv6(t *testing.T) {
 				},
 			}
 
+			if len(tc.ipFamilyPolicy) > 0 {
+				svc.Spec.IPFamilyPolicy = &tc.ipFamilyPolicy
+			}
```

Comment:
WDYT about changing tc.ipFamilyPolicy to pointer *v1.IPFamilyPolicy type?

Then this check for length will not be needed, and type will be aligned just with Service spec type, so we can assign it straight to spec field

Reply:
that complicates to define the inputs vars, doesn't it? 
you have to declare the vars at the beginning of the test code and then reference them from the input var

Reply:
That could've been a problem if in test cases we were using some not defined strings

But we are anyway using variables like v1.IPFamilyPolicySingleStack, so we can just do &v1.IPFamilyPolicySingleStack?

Reply:
Oh, nevermind, I am wrong, we can not take address of consts, sorry

Reply:
yeah, you can define a variable and reference it, but instead of the `if` here you'll have 3 new variables, 🤷 _tomato, tomatoe_